quickstart-sda.md

```diff
@@ -0,0 +1,239 @@
+
+```bash
+docker cp ingest:/shared/c4gh.pub.pem .
+```
+
+**Encrypt and upload**
+
+To encrypt and upload `test_file` to the s3inbox first get a token and prepare a s3cmd config file as described in the section [Uploading data](#uploading-data) above. Then run the following:
+
+```bash
+./sda-admin --sda-config s3cmd.conf --sda-key c4gh.pub.pem upload test_file
+```
+
+One can verify that the encrypted file is uploaded in the archive's inbox by the following command:
+
+```cmd
+sda-cli list --config s3cmd.conf
+```
+
+**Ingesting**
+
+To list the filenames currently in the "inbox" queue waiting to be ingested run:
+
+```bash
+./sda-admin ingest
+```
+
+If `test_file.c4gh` is in the returned list, run:
+
+```bash
+./sda-admin ingest test_file
+```
+to trigger ingestion of the file.
+
+**Adding accession IDs**
+
+Check that the file has been ingested by listing the filenames currently in the "verified" queue waiting to have accession IDs assigned to them:
+
+```bash
+./sda-admin accession
+```
+
+If `test_file.c4gh` is in the returned list, we can proceed with accession:
+
+```bash
+./sda-admin accession MYID001 test_file
+```
+
+where `MYID001` is the `accession ID` we wish to assign to the file.
+
+**Mapping to datasets**
+
+Check that the file got an accession ID by listing the filenames currently in the "completed" queue waiting to be associated with a dataset ID:
+
+```bash
+./sda-admin dataset
+```
+
+Lastly, associate the file with a dataset ID:
+
+```bash
+./sda-admin dataset MYSET001 test_file
+```
+
+Note that all the above steps can be done for multiple files at a time except from assigning accession IDs which needs to be done for one file at a time.
+
+### Monitoring the status of services
+
+Assuming access to a terminal session in the host machine of the deployed docker compose stack, the status of all running containers can be checked as per usual with the command: `docker ps` whereas all logs from the deployed  services can be monitored in real time as per usual by the command:
+
+```cmd
+docker compose -f docker-compose.yml logs -f
+```
+
+or per service as:
+
+```cmd
+docker compose -f docker-compose.yml logs <container-name> -f
+```
+
+Note that when applicable periodic `healthchecks` are in place to ensure that services are running normally. All containers are configured to always restart upon failure.
+
+### Working with rabbitmq
+
+As stated, we use [RabbitMQ](https://www.rabbitmq.com/) as our message broker between different services in this stack. Monitoring the status of the broker service can most conveniently be done via the web gui which is accesible at: http://localhost:15672/ (use `https` if TLS is enabled). By default, `user:password` credentials with values `test:test` are created upon deployment and can be changed by editing the `docker-compose.yml` file. There are two ways to create a password hash for RabbitMQ as described [here](https://www.rabbitmq.com/passwords.html#computing-password-hash)
```

Comment:
```suggestion
We use [RabbitMQ](https://www.rabbitmq.com/) as our message broker between different services in this stack. Monitoring the status of the broker service can most conveniently be done via the web GUI which is accessible at: http://localhost:15672/ (use `https` if TLS is enabled). By default, `user:password` credentials with values `test:test` are created upon deployment and can be changed by editing the `docker-compose.yml` file. There are two ways to create a password hash for RabbitMQ as described [here](https://www.rabbitmq.com/passwords.html#computing-password-hash)
```

Reply:
The GUI monitoring of the status of MQ, can we only do that from the localhost? How if GUI is not possible for the production server? Or if tunneling is needed, maybe give some instruction about tunneling?

Reply:
That is a valid point. But how to access the GUI depends on system specifics. So, I decided to refer only to the simplest case where the deployment is run locally. But I can look into this if we feel it is necessary :-)